backend/pycon/settings/base.py

```diff
@@ -146,3 +148,24 @@
 
 STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
 MAPBOX_PUBLIC_API_KEY = env("MAPBOX_PUBLIC_API_KEY", default="")
+
+USE_AWS_S3 = env("USE_AWS_S3", default=False) == "True"
```

Comment:
Once you are done testing can you move the settings back to prod? If we need to test again we can set `DJANGO_MODULE_SETTINGS` to `pycon.settings.prod` to change the settings that `runserver` loads